salt/utils/sanitisers.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
```

Comment:
AMERICA!
Normally we stick with American spelling, just because originally this is how it was all spelled, no offense intended....